APIGOV-18093 - note about catalog subscription mode (#1988)
Axway/axway-open-docs: content/en/docs/central/connect-api-manager/subscription-for-the-consumer.md

CENTRAL_SUBSCRIPTIONS_APPROVAL_WEBHOOK_URL={The webhook URL that subscription data will be posted to}
CENTRAL_SUBSCRIPTIONS_APPROVAL_WEBHOOK_HEADERS={The headers that will be used when posting data to the webhook url}
```

> {{< alert title="Note" color="primary" >}}Each API in Axway API Manager that is discovered will have the subscription approval mode for its corresponding catalog item in Amplify Central set to whatever the configured mode is at the time the API is discovered and published. Changing the configured mode and restarting the Discovery agent will not change the mode of any existing Amplify Central catalog items. To get the newly configured mode to be changed in any catalog items, the APIs in Axway API Manager must be rediscovered.{{< /alert >}}

Author: hmm. OK. Will fix.

> {{< alert title="Note" color="primary" >}}Each API in Axway API Manager that is discovered will have the subscription approval mode for its corresponding catalog item in Amplify Central set to whatever the configured mode is at the time the API is discovered and published. Changing the configured mode and restarting the Discovery agent will not change the mode of any existing Amplify Central catalog items. To get the newly configured mode to be changed in any catalog items, the APIs in Axway API Manager must be rediscovered.{{< /alert >}}

Author: Odd. In netlify or VS code, I don't see this char. I even copy/paste this paragraph into the other 2 files and the char doesn't show up there.

> {{< alert title="Note" color="primary" >}}Each API in Axway API Manager that is discovered will have the subscription approval mode for its corresponding catalog item in Amplify Central set to whatever the configured mode is at the time the API is discovered and published. Changing the configured mode and restarting the Discovery agent will not change the mode of any existing Amplify Central catalog items. To get the newly configured mode to be changed in any catalog items, the APIs in Axway API Manager must be rediscovered.{{< /alert >}}

Reviewer: ```suggestion

{{< alert title="Note" color="primary" >}}Each API that is discovered in Axway API Manager contains the subscription approval mode for its corresponding Amplify Central catalog item, which is set when the API is discovered and published. The only way to change a catalog item's subscription approval mode is to rediscover the API in Axway API Manager.{{< /alert >}}
```

> {{< alert title="Note" color="primary" >}}Each API in Axway API Manager that is discovered will have the subscription approval mode for its corresponding catalog item in Amplify Central set to whatever the configured mode is at the time the API is discovered and published. Changing the configured mode and restarting the Discovery agent will not change the mode of any existing Amplify Central catalog items. To get the newly configured mode to be changed in any catalog items, the APIs in Axway API Manager must be rediscovered.{{< /alert >}}

Reviewer: Please update all three notes. Cyrille has already approved the wording.

> {{< alert title="Note" color="primary" >}}Each API in Axway API Manager that is discovered will have the subscription approval mode for its corresponding catalog item in Amplify Central set to whatever the configured mode is at the time the API is discovered and published. Changing the configured mode and restarting the Discovery agent will not change the mode of any existing Amplify Central catalog items. To get the newly configured mode to be changed in any catalog items, the APIs in Axway API Manager must be rediscovered.{{< /alert >}}

Reviewer: Use this wording but tweak for AWS API Gateway and Azure API Gateway:
Each API that is discovered in Axway API Manager contains the subscription approval mode for its corresponding Amplify Central catalog item, which is set when the API is discovered and published. The only way to change a catalog item's subscription approval mode is to update the agent configuration mode (refer to `CENTRAL_SUBSCRIPTIONS_APPROVAL_MODE`), restart the agent, and then rediscover the API in Axway API Manager.

> {{< alert title="Note" color="primary" >}}Each API in Axway API Manager that is discovered will have the subscription approval mode for its corresponding catalog item in Amplify Central set to whatever the configured mode is at the time the API is discovered and published. Changing the configured mode and restarting the Discovery agent will not change the mode of any existing Amplify Central catalog items. To get the newly configured mode to be changed in any catalog items, the APIs in Axway API Manager must be rediscovered.{{< /alert >}}

Author: looks good
